assets/src/apps/delivery/components/ActivityRenderer.tsx

```diff
@@ -327,12 +374,48 @@ const ActivityRenderer: React.FC<ActivityRendererProps> = ({
   const currentActivityId = useSelector(selectCurrentActivityId);
   const initPhaseComplete = useSelector(selectInitPhaseComplete);
   const initStateFacts = useSelector(selectInitStateFacts);
+
+  const updateGlobalState = async (snapshot: any, stateFacts: any) => {
+    const payloadData = {} as any;
+    stateFacts.map((fact: string) => {
+      // EverApp Information
+      if (fact.startsWith('app.')) {
+        const data = fact.split('.');
+        const objId = data.splice(2).join('.');
+        const value = snapshot[fact];
+        payloadData[data[1]] = { ...payloadData[data[1]], [objId]: value };
+      }
+    });
+
+    const arrayPLData = Object.keys(payloadData);
+    const retrivedData = (await Extrinsic.readGlobal(arrayPLData)) as any;
```

Comment:
typo in 'retrievedData' and 'parsedRetrievedData'

Reply:
addressed.